src/forklift/engine.py

```diff
@@ -148,7 +149,9 @@ def lift_pallets(file_path=None, pallet_arg=None, skip_git=False):
 
     start_process = clock()
     core.init(log)
-    lift.process_crates_for(pallets_to_lift, core.update)
+
+    change_detection = ChangeDetection(config.get_config_prop('changeDetectionTables'), dirname(config_location))
```

Comment:
if you if checked the `changeDetectionTables` config prop here, setting it to a default `[]` if it wasn't found, would the breaking change still exist?

Reply:
Good catch. I believe that it would remove the breaking change. See: 6757fdb